src/modules/capabilities.json

```diff
@@ -0,0 +1,718 @@
+                            }
+                        ]
+                    }
+                }
+            ]
+        },
+        {
+            "name": "request",
+            "summary": "Requests grants for all capability/role combinations in the roles array.",
+            "params": [
+                {
+                    "name": "grants",
+                    "required": true,
+                    "schema": {
+                        "type": "array",
+                        "items": {
+                            "$ref": "https://meta.comcast.com/firebolt/capabilities#/definitions/Permission"
+                        },
+                        "minItems": 1
+                    }
+                }
+            ],
+            "result": {
+                "name": "Default Result",
+                "summary": "Returns an array of CapabilityInfo objects for the passed in capabilities.",
+                "schema": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "https://meta.comcast.com/firebolt/capabilities#/definitions/CapabilityInfo"
+                    },
+                    "minItems": 1
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default result",
+                    "params": [
+                        {
+                            "name": "grants",
+                            "value": [
+                                {
+                                    "role": "use",
+                                    "capability": "xrn:firebolt:capability:commerce:purchase"
+                                }
+                            ]
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": [
+                            {
+                                "capability": "xrn:firebolt:capability:commerce:purchase",
+                                "role": "use",
+                                "supported": true,
+                                "available": true,
+                                "permitted": true,
+                                "granted": true
+                            }
+                        ]
+                    }
+                }
+            ]
+        },
+        {
+            "name": "onAvailable",
+            "tags": [
+                {
+                    "name": "event"
+                }
+            ],
+            "summary": "Listens for all App permitted capabilities to become available.",
+            "params": [
+                {
+                    "name": "capability",
+                    "required": true,
+                    "schema": {
+                        "$ref": "https://meta.comcast.com/firebolt/capabilities#/definitions/Capability"
+                    }
+                },
+                {
+                    "name": "role",
```

Comment:
Kevin pointed out that available/unavailable is not specific to `role` so we should drop that param. I just updated the spec.